src/components/main/menu/DateSelector/DateRangePicker.jsx

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'proptypes';
+import moment from 'moment';
+import DatePicker from 'react-datepicker';
+import Button from '../../../common/Button';
+
+import {
+  updateStartDate,
+  updateEndDate,
+  closeDateRangeModal,
+} from '../../../../redux/reducers/data';
+
+import 'react-datepicker/dist/react-datepicker.css';
+import COLORS from '../../../../styles/COLORS';
+
+const cardStyle = {
+  height: '299px',
+  width: '400px',
+  overflow: 'visible',
+  boxShadow: '0px 6px 5px rgba(0, 0, 0, 0.5)',
+};
+
+const headerStyle = {
+  height: '50px',
+  background: COLORS.BACKGROUND,
+  color: COLORS.FONTS,
+  fontWeight: 'bold',
+  fontSize: '20px',
+  border: 'none',
+  borderRadius: '0',
+};
+
+const textStyle = {
+  fontSize: '16px',
+};
+
+const inputSpanStyle = {
+  display: 'inline-block',
+  width: '100px',
+  color: headerStyle.color,
+  ...textStyle,
+};
+
+const containerDivStyle = {
+  height: '42px',
+  width: '317px',
+  display: 'flex',
+  alignItems: 'center',
+};
+
+const DateRangePicker = ({
+  id,
+  title,
+  style,
+  closeModal,
+  updateStart,
+  updateEnd,
+}) => {
+  const [startDate, updateLocalStart] = useState();
+  const [endDate, updateLocalEnd] = useState();
+
+  const handleDateChange = (updateStartOrEndDate, date) => {
+    updateStartOrEndDate(date);
+  };
+
+  return (
+    <div
+      id={id}
+      className="modal-card"
+      style={{ ...cardStyle, ...style }}
+    >
+      {/* ---------- Modal Card Header ---------- */}
+      <header
+        className="modal-card-head"
+        style={headerStyle}
+      >
+        <p
+          className="modal-card-title has-text-weight-bold is-size-5"
+          style={{
+            color: headerStyle.color,
+            paddingLeft: '18px',
+            paddingTop: '18px',
+          }}
+        >
+          {title}
+        </p>
+        <button
```

Comment:
Can this be a button component?

Reply:
This is the 'X' to close the rangepicker modal. Replaced with the `<Icon />` component wrapped in an anchor tag to handle cursor change on hover.

Reply:
Just saw in your PR that you added icon to the Button component. I'll look at swapping this out after your PR is merged in.